2017-09-15T18:02:34.840+0300 I CONTROL  [initandlisten] MongoDB starting : pid=15776 port=27017 dbpath=/Users/panda/Work/glu_mobile_test/data/db 64-bit host=Sergeys-MBP
2017-09-15T18:02:34.841+0300 I CONTROL  [initandlisten] db version v3.4.9
2017-09-15T18:02:34.841+0300 I CONTROL  [initandlisten] git version: 876ebee8c7dd0e2d992f36a848ff4dc50ee6603e
2017-09-15T18:02:34.841+0300 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2l  25 May 2017
2017-09-15T18:02:34.841+0300 I CONTROL  [initandlisten] allocator: system
2017-09-15T18:02:34.841+0300 I CONTROL  [initandlisten] modules: none
2017-09-15T18:02:34.841+0300 I CONTROL  [initandlisten] build environment:
2017-09-15T18:02:34.841+0300 I CONTROL  [initandlisten]     distarch: x86_64
2017-09-15T18:02:34.841+0300 I CONTROL  [initandlisten]     target_arch: x86_64
2017-09-15T18:02:34.841+0300 I CONTROL  [initandlisten] options: { config: "mongod.conf", net: { bindIp: "127.0.0.1" }, storage: { dbPath: "/Users/panda/Work/glu_mobile_test/data/db", directoryPerDB: true }, systemLog: { destination: "file", logAppend: true, path: "/Users/panda/Work/glu_mobile_test/data/log/mongodb.log" } }
2017-09-15T18:02:34.841+0300 I STORAGE  [initandlisten] exception in initAndListen: 98 Unable to lock file: /Users/panda/Work/glu_mobile_test/data/db/mongod.lock Resource temporarily unavailable. Is a mongod instance already running?, terminating
2017-09-15T18:02:34.841+0300 I NETWORK  [initandlisten] shutdown: going to close listening sockets...
2017-09-15T18:02:34.841+0300 I NETWORK  [initandlisten] shutdown: going to flush diaglog...
2017-09-15T18:02:34.841+0300 I CONTROL  [initandlisten] now exiting
2017-09-15T18:02:34.841+0300 I CONTROL  [initandlisten] shutting down with code:100
